odl/phantom/geometric.py

```diff
@@ -542,36 +570,57 @@ def _ellipsoid_phantom_3d(space, ellipsoids):
     return space.element(p)
 
 
-def ellipsoid_phantom(space, ellipsoids):
+def ellipsoid_phantom(space, ellipsoids, min_pt=None, max_pt=None):
     """Return a phantom given by ellipsoids.
 
     Parameters
     ----------
     space : `DiscreteLp`
-        Space in which the phantom is created, must be 2- or 3-dimensional.
-        If ``space.shape`` is 1 in an axis, a corresponding slice of the
-        phantom is created.
+        Space in which the phantom should be created, must be 2- or
+        3-dimensional. If ``space.shape`` is 1 in an axis, a corresponding
+        slice of the phantom is created (instead of squashing the whole
+        phantom into the slice).
     ellipsoids : sequence of sequences
-        If ``space`` is 2-dimensional each row should contain:
+        If ``space`` is 2-dimensional, each row should contain the entries ::
+
+            'value',
+            'axis_1', 'axis_2',
+            'center_x', 'center_y',
+            'rotation'
+
+        If ``space`` is 3-dimensional, each row should contain the entries ::
+
+            'value',
+            'axis_1', 'axis_2', 'axis_3',
+            'center_x', 'center_y', 'center_z',
+            'rotation_phi', 'rotation_theta', 'rotation_psi'
 
-        'value', 'axis_1', 'axis_2', 'center_x', 'center_y', 'rotation'
+        The provided ellipsoids need to be specified relative to the
+        reference rectangle ``[-1, -1] x [1, 1]``, or analogously in 3d.
+        The angles are to be given in radians.
 
-        If ``space`` is 3-dimensional each row should contain:
+    min_pt, max_pt : array-like, optional
```

Comment:
> This is rather weird since all other parameters work in the set [-1, 1]^n, and here we suddenly work in "real" coordinates so to say. This should at least be mentioned.

I don't agree. The `space` is also not relative to [-1, 1]^n, and `min_pt` and `max_pt` explicitly relate to that.